Add count to non-leaves

mkoske opened the pull request.

Commit:
Add count to non-leaves

ghost commented:
I created new pull request since I'm not very good with git.

ghost commented:
Also, the tests. I'm not familiar with those tests you have there. Would it be just as simple as adding the count in parentheses to node text? Like `text: 'a tree node (0)'`, since it has empty node list?

reergymerej commented:
@mkoske I think you're right about the test error.  You should be able to check locally with `npm test`.  I'm not set up to check it myself right now.

Commit:
Make only one react text element

Commit:
Make tests to pass

ghost commented:
Hello, I updated my pull request.

Tests now pass.

Commit:
Add option to collapse non-leaf nodes by default

ghost commented:
Hi! I also added an option to collapse trees by default. But I haven't made tests to pass. The usecase is that if you have many files with multiple todo's in each, you get quite large tree. When non-leafs are collapsed by default, then you can more easily make sense what's going on. And work on one node at the time.

Commit:
Recognize also Windows path separator

When splitting path names on path separator, it needs to be taken into
account that Windows uses different path separator than *nixes.

Commit:
Update README on config section

Config now allows choosing whether the nodes are collapsed by default or not

Commit:
Update README

reergymerej reviewed (commented):
Thanks for the changes!

reergymerej commented:
Did you create your own fork? I'll merge in the changes is you want to reopen the PR.

ghost commented:
Hi! I reopened this. You can merge changes, but there's one problem with the config I added. I don't know how to fix the tests so they pass. Can you help with this? If you have no time to fix it yourself, could you at least point direction so I can fix it?
